ide/coqide/FAQ.md

```diff
@@ -0,0 +1,85 @@
+# CoqIDE FAQ #
+
+### What is CoqIDE?
+
+A powerful graphical interface for Coq. See https://coq.inria.fr for more information.
+
+### How to enable Emacs keybindings?
+
+Insert the following line in your `gtkrc` file:
+```
+gtk-key-theme-name = "Emacs"
+```
+The location of this file is system-dependent. If you're running
+GNOME, you may use the graphical configuration tools.
+
+See also the µPG mode in CoqIDE preferences and help.
+
+### How to enable antialiased fonts?
+
+Set the `GDK_USE_XFT` variable to 1. This is the default with Gtk >= 2.2.
+If some of your fonts are not available, set `GDK_USE_XFT` to 0.
+
+### How to use those Forall (∀) and Exists (∃) pretty symbols?
+
+Thanks to the Notation features in Coq, you just need to insert these
+lines in your Coq Buffer :
+```
+Notation "∀ x : t, P" := (forall x:t, P) (at level 200, x ident).
+Notation "∃ x : t, P" := (exists x:t, P) (at level 200, x ident).
+```
+Copy-pasting of these lines from this file will not work outside of CoqIDE.
+You need to load a file containing these lines or to enter the `∀`
+using an input method (see next question below).
+To try it, just use `Require utf8` from inside CoqIDE.
+
+To enable these notations automatically, start coqide with:
+```
+coqide -l utf8
+```
+
+In the `ide` subdir of Coq library, you will find a sample `utf8.v` with some
+pretty simple notations.
```

Comment:
I'm confused. Is there a file like this?

Reply:
Actually not. I can’t find any on my system.

Reply:
Perhaps  `theories/Unicode/Utf8.v`